velox/exec/fuzzer/PrestoQueryRunner.h

```diff
@@ -60,7 +60,19 @@ class PrestoQueryRunner : public velox::exec::test::ReferenceQueryRunner {
   /// the query must already exist.
   std::vector<velox::RowVectorPtr> execute(const std::string& sql);
 
+  bool supportsVeloxVectorResults() const override;
+
+  std::vector<RowVectorPtr> executeVector(
+      const std::string& sql,
+      const std::vector<RowVectorPtr>& input,
+      const RowTypePtr& resultType) override;
+
  private:
+  std::vector<velox::RowVectorPtr> executeImpl(
```

Comment:
Taking a closer look, it appear that this executeImpl method is not needed at all. You can just implement executeVector and call it from 'execute'.